packages/blade/src/components/Button/BaseButton/getBaseButtonStyles.ts

```diff
@@ -0,0 +1,31 @@
+import type { StyledBaseButtonProps } from './StyledBaseButton';
+
+const getBaseButtonStyles = ({
+  color,
+  hoverColor,
+  activeColor,
+}: Omit<StyledBaseButtonProps, 'children' | 'onClick'>): string => `
+  display:flex;
+  flex-direction: row;
+  height: 48px;
+  cursor: pointer;
+  background-color: ${color};
+  border: 2px solid ${color};
+  color: white;
+  border-radius: 2px;
+  justify-content: center;
+  align-items: center;
+  gap: 8px;
+  align-self: center;
+  padding: 0px 20px 0px 20px;
+  &:hover {
+    background-color: ${hoverColor};
+    border: 2px solid ${hoverColor};
+  }
+  &:active {
```

Comment:
what happens to these pseudo selectors on native? they don't break?

Reply:
They get ignored

Reply:
What is the alternative in react native then? There is no "active" state?

Reply:
There is an active state, it's when you press the button to indicate it's being pressed. There is no focus state though. The `pressed` state we have to handle in `StyledBaseButton.native.tsx` since that's the only way they've enabled it. Ref: https://reactnative.dev/docs/pressable#style